AlphaWallet/Market/Coordinators/UniversalLinkCoordinator.swift

```diff
@@ -127,8 +130,16 @@ class UniversalLinkCoordinator: Coordinator {
         if isForTransfer {
             parameters.removeValue(forKey: "price")
         }
+
+        if signedOrder.order.spawnable {
```

Comment:
It looks like we will no longer hit `Constants.currencyDropServer` when `signedOrder.order.nativeCurrencyDrop` == true. Is this what we want?

If not, might be better to restore the assignment for `query` back into `getParametersAndQuery()` and add a check for `signedOrder.order.spawnable` there and assign accordingly.

Reply:
@hboon thats handled by createHTTPParametersForCurrencyLinksToPaymentServer in the switch statement?

Reply:
Ah, right.